examples/utils/embeddings_utils.py

```diff
@@ -9,58 +9,57 @@
 from sklearn.metrics import average_precision_score, precision_recall_curve
 from tenacity import retry, stop_after_attempt, wait_random_exponential
 
-import openai
+from openai import openAI
```

Comment:
@ARajgor For me this yields and import error:
> ImportError: cannot import name 'openAI' from 'openai'

Should this OpenAI with a capital O